iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -897,17 +898,31 @@ static bool isFusableWithConsumer(OpOperand &use) {
   return areFusableLinalgOps(use);
 }
 
+/// For all uses of an operation, finds the use that dominates all other uses.
+static Optional<OpOperand *> getFusableUse(Operation *op,
+                                           DominanceInfo &dominanceInfo) {
+  for (auto &use : op->getUses()) {
```

Comment:
I wonder if we can simply use getUsers()?

Reply:
The method returns the `use`. So iterating over uses makes more sense.